specifyweb/frontend/js_src/lib/components/AppResources/Editor.tsx

```diff
@@ -124,19 +124,31 @@ export function AppResourceEditor({
         data={resourceData?.data ?? ''}
         resource={resource}
       />
-    </>
+    </div>
   );
 
   return typeof resourceData === 'object' ? (
-    <Container.Base className="flex-1 overflow-hidden">
+    <Container.Base className="min-h-[30vh] flex-1 overflow-auto">
       <DataEntry.Header>
-        {appResourceIcon(getResourceType(resource))}
-        <h3 className="overflow-auto whitespace-nowrap text-2xl">
-          {formatted}
-        </h3>
+        <div className="flex items-center justify-center gap-2">
+          <div className="hidden md:block">
+            {appResourceIcon(getResourceType(resource))}
+          </div>
+          <div className="flex max-w-[90%] gap-1">
+            <h3 className="overflow-auto whitespace-nowrap text-2xl">
+              {formatted}
+            </h3>
+            <AppResourceEditButton title={title}>
+              {form()}
+            </AppResourceEditButton>
+          </div>
+        </div>
         {headerButtons}
       </DataEntry.Header>
-      <Form className="flex-1 overflow-hidden" forwardRef={setForm}>
+      <Form
+        className="max-h-[50vh] max-h-[90%] min-h-[30vh] flex-1 overflow-auto"
```

Comment:
You have max-h twice - redundant.
Also, why do you need both max-h and min-h?

Reply:
It is for the mobile version, it doesn't influence larger screens. See screen shots with and without min-h
![Screenshot 2023-02-17 at 9 05 24 AM](https://user-images.githubusercontent.com/108160931/219717984-546f67da-f942-440f-a0d1-9ce56dfbc44f.png)

Reply:
![Screenshot 2023-02-17 at 9 05 37 AM](https://user-images.githubusercontent.com/108160931/219718036-56ffaf19-39f5-4816-9a28-e7a46e38740c.png)